### PR TITLE
Added linter

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: david-doit-intl <david@doit-intl.com>

### CODE REVIEW

**Comment by david-doit-intl** [comment]
Added Black which is a code linter.
To run, 
```
poetry run black .
```

File: compile.py [hunk 1]
```diff
@@ -40,140 +40,183 @@ def create_view(dataset_name, view_name, project, viewSQL):
...
         deploy_all(args.dataset)
 
     with indent(4, quote="* "):
         puts(colored.yellow("Finished!"))
-      
+
+
 if __name__ == "__main__":
-   main()
+    main()
```
Comment (ebendutoit) [inline compile.py]: Is the indentation wrong here?

File: poetry.lock [hunk 2]
```diff
@@ -14,6 +22,40 @@ optional = false
...
+[[package]]
+category = "dev"
+description = "Classes Without Boilerplate"
+name = "attrs"
+optional = false
+python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
+version = "19.3.0"
+
+[package.extras]
+azure-pipelines = ["coverage", "hypothesis", "pympler", "pytest (>=4.3.0)", "six", "zope.interface", "pytest-azurepipelines"]
```
Comment (ebendutoit) [inline poetry.lock]: Do we need all the stuff in the lock file?

Comment (david-doit-intl) [reply poetry.lock]: Yes otherwise people do not use the same packages.

File: compile.py (hunk 1 above)
Comment (david-doit-intl) [reply compile.py]: No, 4 spaces. Looks like you had 3 before.

**Comment by david-doit-intl** [comment]
@ebendutoit sorry been away for a while, find my answers above!

**Review by ebendutoit — APPROVED** [review]

**Pull request merged**